pallets/offchain-worker/src/test_data.md

```diff
@@ -1,12 +1,14 @@
 # Ethereum signature for test
 eth address 0x4d88dc5d528a33e4b8be579e9476715f60060582
+[77, 136, 220, 93, 82, 138, 51, 228, 184, 190, 87, 158, 148, 118, 113, 95, 96, 6, 5, 130]
```

Comment:
Maybe have a link point to this where these are used

Reply:
just keep it as notes for manual testing.